mpt_api_client/http/async_client.py

```
@@ -30,19 +30,19 @@ def __init__(
         timeout: float = 20.0,
         retries: int = 5,
     ):
-        api_token = api_token or os.getenv("MPT_TOKEN")
+        api_token = api_token or os.getenv("MPT_API_TOKEN")
         if not api_token:
             raise ValueError(
                 "API token is required. "
-                "Set it up as env variable MPT_TOKEN or pass it as `api_token` "
+                "Set it up as env variable MPT_API_TOKEN or pass it as `api_token` "
                 "argument to MPTClient."
             )
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)